src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -340,6 +340,11 @@ private void requireDockerApiVersionAtLeast(final String required, final String
     assumeTrue(msg, dockerApiVersionAtLeast(required));
   }
 
+  private boolean storageDriverNot(String driver) throws Exception {
```

Comment:
Sounds good. Will do it right away so we can merge this ASAP. Thanks for the feedback